src/cargo/ops/registry.rs

```diff
@@ -442,13 +443,26 @@ fn wait_for_publish(
 ) -> CargoResult<()> {
     let version_req = format!("={}", pkg.version());
     let mut source = SourceConfigMap::empty(config)?.load(registry_src, &HashSet::new())?;
-    let source_description = source.describe();
+    // Disable the source's built-in progress bars. Repeatedly showing a bunch
+    // of independent progress bars can be a little confusing. There is an
+    // overall progress bar managed here.
+    source.set_quiet(true);
+    let source_description = source.source_id().to_string();
```

Comment:
What did `source.describe()` to `source.source_id().to_string()` change?

Reply:
`source.describe()` ends up calling `display_index()` which renders as something like `crates.io index`. The SourceId display renders like ``` registry `crates-io` ```. The other form ended up being a little awkward from a grammar perspective with the way I worded it.

Reply:
Thanks for the explanation!  I think in the original version, it was more like it is today but then I had to change how I got my Source and I didn't realize I could do this to get the old behavior.